runtime/development/src/lib.rs

```diff
@@ -1295,6 +1295,46 @@ impl pallet_xcm_transactor::Config for Runtime {
 parameter_types! {
 	pub const MaxActiveLoansPerPool: u32 = 50;
```

Comment:
Unrelated but could we bump this? I think we soon will want to test pools with larger # of assets per pool.

Reply:
What is an ok number here? Maybe 300 or so by now?

Reply:
What's the theoretical upper limit now based on how many loans can fit in say half a block based on the benchmarks?

Assuming the theoretical is now a multiple of 1000, I would probably just set this to something like 1000, at least in the development env, for testing purposes

Reply:
Ok! I do not remember the number well, but once I had the benchmark working I can test it and adjust this number if needed. By now, I set it to 1000 👍🏻